include/aspect/particle/world.h

```diff
@@ -268,10 +269,19 @@ namespace aspect
         ParticleLoadBalancing particle_load_balancing;
 
         /**
-         * Limit for how many particles are allowed per cell. This limit is
+         * Lower limit for particle number per cell. This limit is
+         * useful for adaptive meshes to prevent fine cells from being empty
+         * of particles. It will be checked and enforced after mesh
+         * refinement and after particle movement.
+         */
+        unsigned int min_particles_per_cell;
+
+        /**
+         * Upper limit for particle number per cell. This limit is
          * useful for adaptive meshes to prevent coarse cells from slowing down
-         * the whole model. It will only be checked and enforced during mesh
-         * refinement and MPI transfer of particles.
+         * the whole model. It will be checked and enforced after mesh
+         * refinement, after MPI transfer of particles and after particle
+         * movement.
          */
         unsigned int max_particles_per_cell;
```

Comment:
Out of curiosity, the documentation doesn't say what happens when the maximum is exceeded. Would it be worth saying what happens in that case here in the documentation?